pkg/csi/cinder/openstack/openstack_test.go

```diff
@@ -76,10 +77,41 @@ rescan-on-resize=true`
 	expectedOpts.Global.Region = fakeRegion
 	expectedOpts.BlockStorage.RescanOnResize = true
 
-	// Invoke GetConfigFromFile
-	actualAuthOpts, err := GetConfigFromFile(fakeFileName)
+	// Invoke GetConfigFromFiles
+	actualAuthOpts, err := GetConfigFromFiles([]string{fakeFileName})
 	if err != nil {
-		t.Errorf("failed to GetConfigFromFile: %v", err)
+		t.Errorf("failed to GetConfigFromFiles: %v", err)
+	}
+
+	// Assert
+	assert.Equal(expectedOpts, actualAuthOpts)
+
+	// Create an override config file
+	var fakeOverrideFileContent = `
+[BlockStorage]
+rescan-on-resize=false`
+
+	f, err = os.Create(fakeOverrideFileName)
+	if err != nil {
+		t.Errorf("failed to create file: %v", err)
+	}
+
+	_, err = f.WriteString(fakeOverrideFileContent)
+	f.Close()
+	if err != nil {
+		t.Errorf("failed to write file: %v", err)
+	}
+	defer os.Remove(fakeOverrideFileName)
+
+	// expectedOpts should reflect the overriden value of rescan-on-resize. All
```

Comment:
cloud-provider-openstack-format job wants s/overriden/overridden/

Reply:
Thanks, @tombarron ! I have not yet learned how to read that output and had missed the actual error.

Reply:
I wouldn't say I have learned either -- best I know on this log is to look for ERROR then check whether the error is ignored since it may be an expected condition.  The ansible RECAP messages seem to reveal less than with OpenStack logs.